.github/workflows/codeql-codescan.yml

```
@@ -8,17 +8,14 @@
-
 name: CodeQL codescan workflow
-
 on:
   pull_request:
     branches:
       - '**'  
   push:
     branches:
-      - '**'   
-
+      - '**'  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)